### PR TITLE
Update portuguese typos and fixed version

### PR DESCRIPTION
**Pull request by priromagnoli**
Changes in this pull request:

-
-
-

### CODE REVIEW

File: pt/deploy/README.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ SQLite. De qualquer forma, ignorar seu banco de dados SQLite na cópia no GitHub
 criou até agora vão ficar disponíceis apenas localmente, e você terá que adicioná-las novamente em produção. Você deve pensar no seu banco de dados local como um bom "playground", onde você pode testar coisas diferentes e não ter medo de apagar as
```
Comment (GabrielCzar) [inline pt/deploy/README.md]: Corrigir a palavra "disponíceis"

File: pt/deploy/README.md [hunk 2]
```diff
@@ -188,8 +188,8 @@ Assim como fez em seu próprio computador, você pode criar um virtualenv na Pyt
     {% filename %}PythonAnywhere command-line{% endfilename %}
     20:20 ~ $ cd my-first-blog
 
-    20:20 ~ $ virtualenv --python=python3.6 myvenv
-    Running virtualenv with interpreter /usr/bin/python3.6
+    20:20 ~ $ virtualenv --python=python3.5 myvenv
+    Running virtualenv with interpreter /usr/bin/python3.5
```
Comment (aniav) [inline pt/deploy/README.md]: We probably don't want this change

File: pt/deploy/README.md (hunk 1 above)
Comment (rsip22) [reply pt/deploy/README.md]: It was already wrong!

File: pt/deploy/README.md (hunk 2 above)
Comment (rsip22) [reply pt/deploy/README.md]: @priromagnoli Indeed, please change it back.

Comment (rsip22) [reply pt/deploy/README.md]: There is another PR that corrects this, so I'll merge this one and the other afterwards so it will be fixed.

Comment (rsip22) [reply pt/deploy/README.md]: Actually, @aniav, now I get it.

The install instructions in portuguese are in Python 3.5, so this makes sense (unless we change that instead).

Comment (priromagnoli) [reply pt/deploy/README.md]: @rsip22 Yes... This was the reason why I've changed it, Python version is defined as 3.5 in PythonAnywhere. I'm sorry for not explaining this before!

**Pull request merged**